randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+    "323204925": [
+        # Watery Hall Access Wall Tip
+        # Replaces Watery Hall Access holobanner scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;In addition to missiles, the weak wall section in this room is able to be destoyed "
+            "by the &main-color=#FF3333;Charge Beam&main-color=#89D6FF;."
+        ),
+    ],
+    "56434148": [
+        # Central Dynamo Unlocked Door Tip
+        # Replaces Central Dynamo lower right computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        "&main-color=#89D6FF;Fighting the Cloaked Drone locks all doors except the door to Quarantine Access A.",
+    ],
+    "3355561537": [
+        # Central Dynamo IS Tip
+        # Replaces Central Dynamo lower left computer screen scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;A corner of this room can be used to acquire &main-color=#FF3333;Infinite Speed"
+            "&main-color=#89D6FF;. Using Infinite Speed before defeating the Cloaked Drone "
+            "results in taking significantly less damage."
+        ),
+    ],
+    "1755484832": [
+        # Central Dynamo leaving tip
+        # Replaces Central Dynamo upper left computer screen scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;When the Cloaked Drone is defeated, "
+            "it is possible to leave the room without collecting the item."
+        ),
+    ],
+    "2271012115": [
+        # Spinner PB Tip
+        # Replaces Spinner scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Spinner",
+        (
+            "&main-color=#89D6FF;In lieu of Morph Ball Bombs, one can also exit Spinners "
+            "with &main-color=#FF3333;Power Bombs&main-color=#89D6FF;."
+        ),
+    ],
+    "263442559": [
+        # Flaahgra Reverse Kill Tip
+        # Replaces Sunchamber mirror scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;If coming from Sun Tower Access, Flaahgra can be defeated without shooting the first "
+            "mirror.\nScan dashing or &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; "
+            "allows one to reach the center and bomb the bomb slots."
+        ),
+    ],
+    "345113941": [
+        # Metroid Tube Break Tip
+        # Replaces Research Lab Aether top left computer scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;In addition to a missile, a &main-color=#FF3333;Power Bomb&main-color=#89D6FF; or "
+            "&main-color=#FF3333;Plasma Beam&main-color=#89D6FF; shot will break the Metroid's tank."
+        ),
+    ],
+    "437035956": [
+        # HotE Ghost Skip Tip
+        # Replaces HotE statue-side holobanner scans
```

Comment:
This scan node seems too out of the way. I would replace it with the peg scan point and explain that it is used to reach the bomb slot.